projects/shikshalokam/src/lib/core-module/components/side-listing/side-listing.component.html

```diff
@@ -0,0 +1 @@
+<p>side-listing works!</p>
```

Comment:
Why is there no code??

Reply:
if we are passing data from the particular portal itself right ? then again why we need to maintain the listing here